.ci/deps.apt.sh

```diff
@@ -19,7 +19,7 @@ case $CIRCLE_BUILD_IMAGE in
     # Use xenial, needed to replace outdated julia provided by Circle CI
     ADD_APT_UBUNTU_RELEASE=xenial
     # Work around lack of systemd on trusty, which xenial's lxc-common expects
-    echo '#!/bin/sh' | sudo tee /usr/bin/systemd-detect-virt > /dev/null
+    echo '#!/bin/sh' | sudo tee /usr/bin/systemd-detect-virt >/dev/null
```

Comment:
cant we ignore the rule that causes this?

Reply:
Yes.